src/fmeval/eval_algorithms/summarization_accuracy.py

```diff
@@ -77,43 +67,74 @@ class SummarizationAccuracy(EvalAlgorithmInterface):
 
     eval_name = EvalAlgorithm.SUMMARIZATION_ACCURACY.value
 
-    def __init__(
-        self, eval_algorithm_config: SummarizationAccuracyConfig = SummarizationAccuracyConfig(), use_ray: bool = True
-    ):
+    def __init__(self, config: SummarizationAccuracyConfig = SummarizationAccuracyConfig(), use_ray: bool = True):
```

Comment:
Note: for future we should get in a habit of discussing such changes with the whole team. Any direct customer facing changes should first be discussed. In the case of our library it will be the signatures of eval algorithms or model runners.